cmd/apps.go

```diff
@@ -182,7 +182,9 @@ func AppLogs(appID string, lines int) error {
 
 // printLogs prints each log line with a color matched to its category.
 func printLogs(logs string) error {
-	for _, log := range strings.Split(logs, `\n`) {
+	lines := strings.Split(logs, `\n`)
+	// the last line is the string after the final newline and is empty.
+	for _, log := range lines[:len(lines)-1] {
```

Comment:
Perhaps a call to strings.TrimSpace before the Split call would get rid of the empty line :)